packages/snaps-sdk/src/index.ts

```diff
@@ -12,6 +12,9 @@ export {
   typedUnion,
   selectiveUnion,
   nonEmptyRecord,
+  ISO8601DateStruct,
+  ISO8601DurationStruct,
+  toCensoredISO8601String,
```

Comment:
Maybe worth it to only move `ISO8601DateStruct` since we don't need the others? Just to reduce the bloat